plugins/chat/assets/javascripts/discourse/services/chat-composer-presence-manager.js

```diff
@@ -13,7 +13,7 @@ export default class ChatComposerPresenceManager extends Service {
   @service presence;
 
   willDestroy() {
-    this.leave();
+    !isTesting() && this.leave();
```

Comment:
Not 100% sure about this one, but I think there's a race condition in tests, where this will get destroyed sometimes after pretender has been reset and as a result `POST /presence/update` is not handled and causes errors. It became very much visible as Im calling`updatePresence` much more aggressively than before.